app/components/super-features/super-features-relations/super-features-relations-modal/super-features-relations-modal.html

```diff
@@ -0,0 +1,46 @@
+<div id="superFeaturesRelationsModal">
+  <div class="modal-header">
+    <button
+        type="button"
+        class="close"
+        ng-click="$dismiss()">
+      <i class="fa fa-times"></i>
+    </button>
+    <h4 class="modal-relation-type">{{ modalTitle || 'Select the super feature type' }}</h4>
```

Comment:
For the sake of brevity in code, remove these optional text variables:

- `modalTitle`
- `modalBodyBefore`
- `modalBodyAfter`
- `modalOkText`
- `modalCancelText`

And remove the associated directive js and tests.

Unless we plan on reusing this modal and customizing them in the future?